modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="lvms-unstalling-lvms-with-web-console_{context}"]
-= Uninstalling {lvms} installed using the OpenShift Web Console
+= Uninstalling {lvms} installed using the web console
```

Comment:
```suggestion
= Uninstalling {lvms} installed using the {product-title} web console
```

Reply:
I have updated the title to "Uninstalling {lvms} using the web console"
As users can install LVMS using both the web console and the CLI. It's not a restriction that this proc is applicable only if LVMS was installed through the web console.